src/helpers/project.js

```diff
@@ -1,15 +1,18 @@
 const awaitProjects = require('./projects');
-const config = require('../config');
 
-async function fetchProject(name) {
+const args = process.argv.slice(2);
+const talkDomain = args[args.length - 1];
```

Comment:
will this error gracefully if it's missing from the cmd / left off?

Reply:
Not gracefully. If you run `yarn build` then each build exits with errors because the project is undefined.

```
$ NODE_ENV=production node --max-old-space-size=${NODE_RAM:-12000} ./node_modules/@11ty/eleventy/cmd.js --input=src/api
$ NODE_ENV=production node --max-old-space-size=${NODE_RAM:-12000} ./node_modules/@11ty/eleventy/cmd.js --input=src/boards
$ NODE_ENV=production node --max-old-space-size=${NODE_RAM:-12000} ./node_modules/@11ty/eleventy/cmd.js --input=src/tags
$ NODE_ENV=production node --max-old-space-size=${NODE_RAM:-12000} ./node_modules/@11ty/eleventy/cmd.js --input=src/subjects
$ NODE_ENV=production node --max-old-space-size=${NODE_RAM:-12000} ./node_modules/@11ty/eleventy/cmd.js
> Cannot set property 'domain' of undefined

`TypeError` was thrown:
    TypeError: Cannot set property 'domain' of undefined
        at project (/Users/odonnell/zooniverse/Talk-archiver/src/helpers/project.js:15:18)
        at processTicksAndRejections (internal/process/task_queues.js:97:5)
Unhandled rejection in promise ([object Promise]): (more in DEBUG output)
> Cannot set property 'domain' of undefined

`TypeError` was thrown:
    TypeError: Cannot set property 'domain' of undefined
        at project (/Users/odonnell/zooniverse/Talk-archiver/src/helpers/project.js:15:18)
        at processTicksAndRejections (internal/process/task_queues.js:97:5)
Unhandled rejection in promise ([object Promise]): (more in DEBUG output)
> Cannot set property 'domain' of undefined

`TypeError` was thrown:
    TypeError: Cannot set property 'domain' of undefined
        at project (/Users/odonnell/zooniverse/Talk-archiver/src/helpers/project.js:15:18)
        at processTicksAndRejections (internal/process/task_queues.js:97:5)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build:tags" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```